ably/rest_client_test.go

```diff
@@ -21,38 +21,39 @@ import (
 
 var _ = Describe("RestClient", func() {
 	var (
-		server *httptest.Server
+		srv *httptest.Server
 
-		createMockedRestClient = func(server *httptest.Server) *http.Client {
+		newHTTPClientMock = func(srv *httptest.Server) *http.Client {
 			return &http.Client{
 				Transport: &http.Transport{
-					Proxy: func(req *http.Request) (*url.URL, error) {
-						return url.Parse(server.URL)
-					},
+					Proxy: func(req *http.Request) (*url.URL, error) { return url.Parse(srv.URL) },
 				},
 			}
 		}
 	)
 
-	Context("with a failing request", func() {
+	Context("with a failing req", func() {
 		var (
-			statusCode int
-			request    *http.Request
+			req    *http.Request
+			cli    *ably.RestClient
```

Comment:
http://doc.cat-v.org/bell_labs/pikestyle "Variable names"

- `req` is common way of naming `http.Request` variables (https://golang.org/pkg/net/http/)
- `statusCode` the Code part is redundant as the type says its a code (int) and not a status text (string); acutally the variable is removed, but if it'd stayed it'd be names just `status`
- `cli` I agree is artificial, however it conflicts with global `client`, so for readability I named it differently (imo better than client2); suggestions welcome on this one :)